tests/conftest.py

```diff
@@ -28,5 +29,7 @@ def cleanup_children():
             with suppress(psutil.NoSuchProcess):
                 child.kill()
 
-    # Broken code or killing of child processes may leave the MPFuture backend corrupted
-    MPFuture.reset_backend()
+    MPFuture._active_pid = None
```

Comment:
Can we keep the `reset_backend()` method and the comment here, so everything related to `MPFuture` internals is located in `mpfuture.py`?

Reply:
done